backend/typescript/services/implementations/campService.ts

```diff
@@ -68,6 +66,33 @@ class CampService implements ICampService {
         return [];
       }
 
+      if (year) {
+        // Filter out camps that have camp sessions in the desired year
+        /* eslint-disable no-param-reassign */
+        camps = camps.filter((camp) => {
+          // Go through every camp session in a camp. If it has the desired year, include it in the results.
```

Comment:
can we reword this comment- it seems a bit misleading. Something like "go through every camp session in a camp and only include camps where ALL sessions are in the given year"

Reply:
done!